UI/window-basic-main.cpp

```diff
@@ -2670,6 +2670,20 @@ void OBSBasic::CreateHotkeys()
 	LoadHotkeyPair(replayBufHotkeys, "OBSBasic.StartReplayBuffer",
 		       "OBSBasic.StopReplayBuffer");
 
+	auto rb_sv_cb = [](void *data, obs_hotkey_id, obs_hotkey_t *,
```

Comment:
Use a meaningful variable name. Also we prefer camelCase in C++ and snake_case in C.